express/code/blocks/pricing-table/pricing-table.js

```diff
@@ -32,7 +35,11 @@ function handleToggleMore(btn) {
   }
 }
 
-function handleHeading(headingRow, headingCols) {
+function getHeaderId(el) {
+  return el.querySelector('p')?.textContent?.trim().replaceAll(' ', '-') || el.textContent?.trim().replaceAll(' ', '-');
```

Comment:
just for cleaner use of optional chaining, `.textContent` almost always returns a string, so the question mark is not needed after textContent.
Or you can make the whole thing like ```
return (el.querySelector('p')?.textContent || el.textContent || '')
  .trim()
  .replaceAll(' ', '-');
```